core/src/test/java/be/orbinson/aem/dictionarytranslator/servlets/action/ExportDictionaryServletTest.java

```diff
@@ -0,0 +1,109 @@
+package be.orbinson.aem.dictionarytranslator.servlets.action;
+
+import com.day.cq.commons.jcr.JcrConstants;
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+
+import javax.jcr.Session;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(AemContextExtension.class)
+class ExportDictionaryServletTest {
+
+    private final AemContext context = new AemContext();
+    private ExportDictionaryServlet exportTranslation;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        exportTranslation = new ExportDictionaryServlet();
+
+        context.request().setMethod("POST");
+        context.response().setCharacterEncoding("UTF-8");
+        context.create().resource("/test/path");
+        context.registerAdapter(ResourceResolver.class, Session.class, new Function<ResourceResolver, Session>() {
```

Comment:
why do you need this?

Reply:
It is used by the createLanguageResource